dizoo/minigrid/envs/minigrid_wrapper.py

```diff
@@ -32,3 +33,72 @@ def observation(self, obs):
         # print('vis_mask:' + vis_mask)
         image = grid.encode(vis_mask)
         return {**obs, "image": image}
+
+
+class ObsPlusPrevActRewWrapper(gym.Wrapper):
```

Comment:
why add this wrapper here, rather than use the wrapper in `ding/envs`

Reply:
because in `ding/envs`, we use gym for the wrapper, but for minigrid we need gymnasium instead of gym. And in order to make a terrible influence on other env,  I add this wrapper to minigrid wrapper.